mmcls/__init__.py

```diff
@@ -5,8 +5,8 @@
 
 from .version import __version__
 
-mmcv_minimum_version = '1.4.2'
-mmcv_maximum_version = '1.6.0'
+mmcv_minimum_version = '2.0.0rc0'
+mmcv_maximum_version = '2.0.0'
```

Comment:
is extra version buffer needed here? e.g. 2.2.0

Reply:
`rc` will be a long-term tag, it includes all `rc0`, `rc1`, and all.